azure-pipelines.yml

```diff
@@ -31,6 +31,9 @@ variables:
 jobs:
 - template: build-pipeline.yml@templates
   parameters:
+    has_ui_tests: true
+    has_unit_tests: false
+    tk_toolchain_ref: ui_automation_on_azure
```

Comment:
Don't forgot to take this one out before merging either. The branch is currently in code review.